content/docs/guides/polyscale-integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Add the Neon PolyScale integration
+subtitle: Set up a global database cache with Neon's PolyScale integration
+enableTableOfContents: true
+---
+
+[PolyScale](https://docs.polyscale.ai/) allows you to easily distribute and cache your data globally through its low-latency edge network for speedy access to your data from anywhere in the world. With Neon's PolyScale integration, you can create a global cache for your Neon database in a few clicks. No coding or infrastructure changes are required. 
+
+Adding the PolyScale integration to your Neon project automatically creates a global cache and provides you with a PolyScale connection string to use in your application in place of your Neon connection string.
```

Comment:
Can you add some notes, that it can be used in "cross-regional replication" use cases?

Reply:
Thank you. I've integrated this into the introduction.